test/distributed/org/apache/cassandra/distributed/test/accord/NewSchemaTest.java

```diff
@@ -75,7 +75,7 @@ private static Query read(String ks, String table, List<ByteBuffer> keys)
         assert !keys.isEmpty();
         Txn.Builder builder = new Txn.Builder();
         for (int i = 0; i < keys.size(); i++)
-            builder.addLet("row" + i, new Select.Builder().withWildcard().withTable(ks, table).withColumnEquals("pk", keys.get(i)));
+            builder.addLet("row" + i, new Select.Builder().withWildcard().withTable(ks, table).value("pk", keys.get(i)));
```

Comment:
in order to make select/update/delete (places that support `WHERE` clause) consistent all `WHERE` based methods are now an interface and each builder implements... this is why this method got moved.

`value` might not be the best name, open to feedback to help readability... maybe `equals`?